pkg/agent/route/route_windows.go

```diff
@@ -113,6 +133,9 @@ func (c *Client) initServiceIPRoutes() error {
 		if err := c.addVirtualServiceIPRoute(false); err != nil {
 			return err
 		}
+		if err := c.addVirtualNodePortDNATIPRoute(false); err != nil {
```

Comment:
is this a step extracted from `addVirtualServiceIPRoute`? If yes, comments of the latter should be updated

Reply:
Thanks for reminding, I'll update it.